### PR TITLE
DOCS-673: fix executable path in SLAM docs

### PR DESCRIPTION
**Pull request by sguequierre**
* Fixed issue on linux tabs
* macOS needs SME validation, this is what I found when I tested on MacOS

### CODE REVIEW

File: docs/program/extend/modular-resources/add-rplidar-module.md [hunk 1]
```diff
@@ -78,6 +78,12 @@ Now, add the Rplidar as a modular component of your robot in the [Viam app](http
 
   ```json
   {
+    "modules": [
+      {
+        "executable_path": "/Users/<YOUR_USERNAME>/Library/Caches/Homebrew/rplidar-module--git",
```
Comment (kkufieta) [inline docs/program/extend/modular-resources/add-rplidar-module.md:83]: This seems odd to me. For example, I don't have that file on my computer (macOS x86_64) even though I have the rplidar-module installed. I also just confirmed that running `brew install rplidar-module` installs the module at `/usr/local/bin/rplidar-module`. 

Why are you suggesting this change?

Comment (sguequierre) [reply docs/program/extend/modular-resources/add-rplidar-module.md:83]: @kkufieta No need, this is inaccurate-- Will keep at usr/local/bin/rplidar-module

File: docs/services/slam/_index.md [hunk 2]
```diff
@@ -215,6 +215,12 @@ To run Cartographer in live mode, follow [these instructions](../../program/exte
 
 ``` json
 {
+  "modules": [
+    {
+      "executable_path": "/Users/<YOUR_USERNAME>/Library/Caches/Homebrew/rplidar-module--git",
```
Comment (sguequierre) [inline docs/services/slam/_index.md:220]:
```suggestion
      "executable_path": "/usr/local/bin/rplidar-module",
```

File: docs/services/slam/run-slam-cartographer.md [hunk 3]
```diff
@@ -195,6 +195,12 @@ At this point, your complete configuration should look like:
 
   ``` json
   {
+    "modules": [
+      {
+        "executable_path": "/Users/<YOUR_USERNAME>/Library/Caches/Homebrew/rplidar-module--git",
```
Comment (sguequierre) [inline docs/services/slam/run-slam-cartographer.md:200]:
```suggestion
        "executable_path": "/usr/local/bin/rplidar-module",
```

File: docs/program/extend/modular-resources/add-rplidar-module.md (hunk 1 above)
Comment (sguequierre) [reply docs/program/extend/modular-resources/add-rplidar-module.md:83]: Thanks for jumping on this so quickly-- I could only find reference to the module on my local machine in the brew Caches when testing brew install on my own, so I wanted to get review and check where it should be. 
I still don't have the module at /usr/local/bin, which is odd

Comment (kkufieta) [reply docs/program/extend/modular-resources/add-rplidar-module.md]: That's weird, indeed. If you'd like to, we can look at this together tomorrow!

**Review by kkufieta — APPROVED** [review]
LGTM!

**Pull request merged**